### PR TITLE
0.27.2 RC2

### PR DESCRIPTION
**Pull request by clanmills**
I'll remove _(WIP)_ when ready for review and tagging.

### CODE REVIEW

**Comment by clanmills** [comment]
Gentlemen.  Please review.

When you've merged, could you tag this as **v0.27.2-RC2** please?  I'll do the builds and publish on https://pre-release.exiv2.org.  We're getting polished and quick now with "the dots".

File: releasenotes/MinGW/ReadMe.txt [hunk 1]
```diff
@@ -43,17 +43,12 @@ To compile and link your own code using installed library and include files
 Method 1: Explicitly set include and linking options
 $ cd <bundle>
 $ g++ -std=c++98 samples/exifprint.cpp -I/usr/local/include -L/usr/local/lib -lexiv2 -o exifprint
-$ export LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
+$ export PATH="/usr/local/bin:$PATH"
 $ ./exifprint --version
```
Comment (piponazo) [inline releasenotes/MinGW/ReadMe.txt]: [suggestion] Those instructions are about compiling and linking, so we could drop the lines which are calling the executable `exifprint`.

File: releasenotes/CYGWIN/ReadMe.txt [hunk 2]
```diff
@@ -40,7 +40,7 @@ To compile and link your own code using installed library and include files
 Method 1: Explicitly set include and linking options
 $ cd <bundle>
 $ g++ -std=gnu++98 samples/exifprint.cpp -I/usr/local/include -L/usr/local/lib -lexiv2 -o exifprint
-$ export LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
+$ export PATH="/usr/local/bin:$PATH"
```
Comment (piponazo) [inline releasenotes/CYGWIN/ReadMe.txt]: Good catch!

File: releasenotes/releasenotes.txt [hunk 3]
```diff
@@ -1,31 +1,66 @@
...
+3) Support for Nikon/AutoFocus and Sony/FocusPosition Metadata
+4) Documentation and man page revisions
+5) Updated Catalan Localisation
+6) Monitoring API changes for v0.27 dot releases
+7) Using mergify to sync select PRs between 0.27-maintenance and 0.28
+
+Issues fixed since v0.27.1
+--------------------------
+
+#928 Corrections to README.md 2.8.Localisation
```
Comment (piponazo) [inline releasenotes/releasenotes.txt]: This is somehow a duplication with what we have in `doc/ChangeLog`. I would just keep the list of solved issues in one place. IMHO, here it is enough to leave the previous lines:

```
Issues Open   for v0.27.2: https://github.com/Exiv2/exiv2/milestone/4	
Issues Closed for v0.27.2: https://github.com/Exiv2/exiv2/milestone/4?closed=1	
```

**Review by piponazo — COMMENTED** [review]
Thanks for taking care of this! I made some suggestions in the PR

File: releasenotes/MinGW/ReadMe.txt (hunk 1 above)
Comment (clanmills) [reply releasenotes/MinGW/ReadMe.txt]: Luis.  Those commands are to compile exifprint locally.  It's "proof that you can compile, link and run an application.  However, the user isn't running bin/exivprint from the bundle.

File: releasenotes/releasenotes.txt (hunk 3 above)
Comment (clanmills) [reply releasenotes/releasenotes.txt]: I agree.  I've changed it.

About ChangeLog:  I had a script in Redmine to produce the report in ChangeLog and haven't investigated how to generate that list from GitHub.  So, it's a cut'n'paste/edit job from the web-site.  I think it's useful to have a summary.  GitHub believes there are 55 closed issues.  I think there are 35 different issues.  About 20 "to_master" duplicates.

Maybe I just stop spending time on that.  Nobody reads it!

So happy about v0.27.2  Kevin did great security work, you and Dan have done good things and our release process is becoming smoother.   Releasing v0.26 was hell.  v0.27 was tough.  v0.27.1 was OK'ish.  v0.27.2 is smooth.  v0.27.3 will be easy.

File: releasenotes/MinGW/ReadMe.txt (hunk 1 above)
Comment (piponazo) [reply releasenotes/MinGW/ReadMe.txt]: Oh I see. Now that I have looked in more detail I see your point. Fine then :wink:

**Review by piponazo — APPROVED** [review]

**Comment by clanmills** [comment]
@D4N  OK for me to "Merge pull request" (=Rebase and Merge)?

Then I can get the builds done and published.  I'll try to figure out how to tag the damn thing.

**Comment by piponazo** [comment]
> @D4N OK for me to "Merge pull request" (=Rebase and Merge)? Then I can get the builds done and published. I'll try to figure out how to tag the damn thing.

Let's merge it so that you can make those builds, and switch your mind into holidays mode :wink: . Afterwards, if @D4N has some feedback we can incorporate his suggestions for the final release.

**Pull request merged**